packages/desktop-client/src/components/mobile/accounts/AccountTransactions.tsx

```
@@ -54,23 +56,41 @@
     [account.id],
   );
 
+  const [showRunningBalances] = useSyncedPref(`show-balances-${account.id}`);
   const [transactionsQuery, setTransactionsQuery] = useState<Query>(
     baseTransactionsQuery(),
   );
+
+  const { isSearching, search: onSearch } = useTransactionsSearch({
+    updateQuery: setTransactionsQuery,
+    resetQuery: () => setTransactionsQuery(baseTransactionsQuery()),
+    dateFormat,
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)